conf.py

```diff
@@ -28,6 +28,7 @@
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 extensions += ['zeek', 'sphinx.ext.todo', 'zeek_pygments', 'literal-emph']
 extensions += ['sphinx.ext.extlinks']
+extensions += ['sphinx_rtd_theme']
```

Comment:
Just for my own curiosity, we don't seem to have this over in Spicy and the build still succeeded (just triggered a test build). Why are we adding this?

Reply:
It's shown in the theme docs: https://sphinx-rtd-theme.readthedocs.io/en/latest/installing.html

But then there is this:

> Adding this theme as an extension is what enables localization of theme strings in your translated output. If these strings are not translated in your output, either we lack the localized strings for your locale, or you are using an old version of the theme.

So, let me drop it given no localization here. Thanks!